common/config.gni

```diff
@@ -58,3 +58,7 @@ if (is_fuchsia && using_fuchsia_sdk) {
 if ((is_ios || is_mac) && defined(enable_bitcode)) {
   flutter_enable_bitcode = enable_bitcode
 }
+
+if (target_cpu == "arm") {
+  feature_defines_list += [ "TARGET_CPU_ARMV7=1" ]
```

Comment:
Do you need to define your own TARGET_CPU_ARMV7? I thought you could reuse existing TARGET_CPU_ARM

Reply:
I didn't see TARGET_CPU_ARM defined anywhere.

Reply:
Well either way it's workign - using that define instead.

Reply:
I take it back, it's not working - that define is there for ARM64 as well

Reply:
I was using `#ifdef` instead of `#if`.  Thanks for the help on this here and offline.